lxc/delete.go

```diff
@@ -6,35 +6,42 @@ import (
 	"os"
 	"strings"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.